rust/template/src/ovsdb_api.rs

```diff
@@ -128,7 +129,9 @@ fn dump_delta(
                 rel.iter()
                     .map(|(v, w)| {
                         assert!(*w == 1);
-                        record_into_insert_str(v.clone().into_record(), table_str)
+                        record_into_insert_str(v.clone().into_record(), table_str, &|t| {
```

Comment:
With the `record_into_insert_str()` change this reference isn't needed
```suggestion
                        record_into_insert_str(v.clone().into_record(), table_str, |t| {
```